src/platforms/ctxlink/WiFi_Server.c

```diff
@@ -641,6 +655,130 @@ bool is_ip_address_assigned(void)
 	return res;
 }
 
+//
+// Format the current connection data for display to user
+//
+//	SSID = 'name'
+//	RSSI = xx
+//	ip = xxx.xxx.xxx.xxx
+//
+void wifi_get_ip_address(char *buffer, uint32_t size)
+{
+	char local_buffer[64] = {0};
+	memset(buffer, 0x00, size - 1);
```

Comment:
`size - 1`? Surely this leaves the last character in the buffer untouched and thus breaks NUL character termination rules.. That said, if correct, please suffix the `1` with a U so the calculation is carried out unsigned for signed-unsigned continuity to `memset()` (3rd parameter is a `size_t`)

Reply:
Fixed.